### PR TITLE
adding mit license

### PR DESCRIPTION
**Pull request by shdblowers**
fixing #10

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 Matt Widman
```
Comment (mgwidmann) [inline LICENSE:3]: my last name is `Widmann` (two `n`s)... Thanks for doing this!

**Comment by mgwidmann** [comment]
I'll just fix it on master! Thanks though!

**Comment by shdblowers** [comment]
@mgwidmann I copied it from your Github profile

**Comment by mgwidmann** [comment]
Doh 😆

**Pull request merged**